sqldelight-compiler/src/test/kotlin/com/squareup/sqldelight/core/QueryWrapperTest.kt

```diff
@@ -308,6 +308,11 @@ class QueryWrapperTest {
   }
 
   @Test fun `queryWrapper generates with migration statements`() {
+    FixtureCompiler.writeSql("""
+      |CREATE TABLE test (
+      |  value1 TEXT
+      |);
+    """.trimMargin(), tempFolder, "0.sqm")
```

Comment:
Is this valid to have `0.sqm`?

Reply:
yes, it's effectively describing how to create the first version of the database (it would never run in practice on an android device, but it would be valid in server world)